Monika After Story/game/zz_calendar.rpy

```diff
@@ -687,6 +690,7 @@ M̼̤̱͇̤ ͈̰̬͈̭ͅw̩̜͇͈ͅa̲̩̭̩ͅs̙ ̣͔͓͚̰h̠̯̫̼͉e̗̗̮r
                                 size=note_text_size,
                                 color=note_color,
                                 outlines=[],
+                                xsize=110,
```

Comment:
Notes must always fit in one line since multiline notes aren't supported. This is redundant at best.

Reply:
I want to change this in a submod. It's easy to change if it's style. But is there any reason not to define it in style?

Reply:
The only outcome of this change you'll get is this.
![Capture](https://user-images.githubusercontent.com/53382877/103495657-989f7480-4e4c-11eb-882a-dd35fc4074ee.PNG)
As you can see the notes overlap each other.

Also, you still need to [rebase](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-put-my-pull-request-in-the-wrong-branch-and-dont-want-to-suffer-the-wrath-of-devs-how-do-i-fix-it) the PR.

Reply:
Ok, I fixed it.